packages/table/src/Table.story.js

```diff
@@ -187,7 +187,24 @@ storiesOf("Table/Basic", module)
             primaryAction={action("primary button clicked")}
             secondaryButtonLabel="View Profile"
             secondaryAction={action("secondary button clicked")}
-          />
+          >
+            <div
+              style={{ display: "grid", gridGap: 8, gridAutoFlow: "column" }}
+            >
+              <Button
+                type={buttonTypes.SECONDARY}
+                onClick={() => action("secondary button clicked")()}
```

Comment:
Currently our story logs the `selectedRows` as well... Maybe we should keep logging it .

No big deal, change if you want to, but this could be written like this: `onClick={action("secondary button clicked")}`

Reply:
I don't get it.
The `children` is `PropTypes.node`. We can't have `selectedRows` anymore.

Reply:
We do have access to `selectedRows` (:

```
               <Button
                  type={buttonTypes.SECONDARY}
                  onClick={() =>
                    action("secondary button clicked")(selectedRows)
                  }
                >
```

![image](https://user-images.githubusercontent.com/17936244/43605382-81ec0020-966e-11e8-96d5-6dfe5d12dff0.png)

Reply:
Oh, ok! We're talking about the story for the `Table` with `SelectionHeader`.
I tought it was the story for `SelectionHeader` alone.

Fixed!